app/models/villa.rb

```diff
@@ -0,0 +1,9 @@
+class Villa < ApplicationRecord
+  belongs_to :user
+  validates :name, presence: true
+  validates :address, presence: true
+  validates :movie_genre, presence: true
```

Comment:
do we already we to add only certain proposition available? eg horror, comedy, ...
maybe we don't need now, but we will need it later, if you don't do it now, then open a task in the backlog so we don't forget

Reply:
i will see how to do it

Reply:
✅